zips/draft-ecc-lockbox-disbursement.md

```diff
@@ -0,0 +1,245 @@
+  height of this ZIP will be usable immediately on activation.
+
+# Specification
+
+This ZIP proposes the creation of a new Zcash Development Fund. The balance of
+this Fund consists of the contents of the ZIP 1015 Deferred Development Fund
+Lockbox as of the activation height of this ZIP, plus any funds that later
+accrue to either the lockbox or to one or more transparent multisig addresses
+as specified by this ZIP.
+
+### One-time lockbox disbursement
+
+The coinbase transaction of the activation block of this ZIP MUST include an
+additional output to a 3-of-5 P2SH multisig with keys held by the following
+"Key-Holder Organizations": Zcash Foundation, the Electric Coin Company,
+Shielded Labs, and two other organizations yet to be decided.
+
+Let $v$ be the zatoshi amount in the Deferred Dev Fund Lockbox at the
+activation height. ($v$ can be predicted in advance given that height.)
+
+The additional coinbase output MUST follow the same consensus rules as apply to
+funding stream outputs [^zip-0207-consensus-rules]. That is, the coinbase
+transaction MUST contain at least one output that pays $v$ zatoshi, in the
+prescribed way defined in ZIP 207, to the above P2SH multisig address. $v$
+zatoshi are added to the transparent transaction value pool to fund this
+output, and subtracted from the balance of the Deferred Dev Fund Lockbox (i.e.
+the latter balance is reset to zero).
+
+Exactly one of the following options will also be taken. The proposal that
+activates this ZIP must define values for the following two parameters:
+
+* $\mathsf{stream\_value}$: the percentage of the block subsidy to send to 
+  a new funding stream, as described in the options below.
+* $\mathsf{stream\_end\_height}$: The ending block height of that stream.
+
+### Option 1: Extend the lockbox funding stream
+
+The ``FS_DEFERRED`` lockbox funding stream is set to receive
+$\mathsf{stream\_value}\%$ of the block subsidy and is extended until block
+height $\mathsf{stream\_end\_height}$ Both of these parameters must must be
+specified by the proposal under which this ZIP is activated.
+
+#### Rationale for Option 1
+
+Performing a one-time disbursement to a P2SH multisig address will provide a
+source of grant funding for a limited period, allowing time for a lockbox
+disbursement mechanism to be specified and deployed, as originally intended by 
+ZIP 1015 [^zip-1015].
+
+In particular, this provides an opportunity for transaction format changes that
+may be required for such a mechanism to be included in the v6 transaction
+format [^zip-0230]. It is desirable to limit the frequency of transaction
+format changes because such changes are disruptive to the ecosystem. It is not
+necessary that protocol rules for disbursement actually be implemented until
+after the transaction format changes are live on the network. It is RECOMMENDED
+that any such transaction format changes be included in the upcoming V6
```

Comment:
"v6" (consistent capitalization).

Reply:
Addressed in https://github.com/zcash/zips/pull/994